charts/renku/values.yaml

```diff
@@ -51,17 +51,17 @@ global:
     ## Renku version to be used (currently for project creation in
     ## the UI) If not set explicitly the version will be picked up
     ## from the respective renku (sub)chart.
-    version: 0.3.0
-  graph:
-    tokenRepository:
-      ## Name of the postgres user to be used to access the db storing access tokens
-      postgresUser: tokenstorage
-      ## Postgres password to be used to access the db storing access tokens
-      ## Generated using: `openssl rand -hex 16`
-      postgresPassword:
-    jena:
-      ## specify which dataset should be used in Jena
-      dataset:
+    version: 0.3.1
+  # graph:
```

Comment:
Should we clean this up or do we plan to plug it back in? :smile:

Reply:
ah thanks, I'll remove the comments